eland/ml/_model_serializer.py

```diff
@@ -94,7 +94,12 @@ def to_dict(self) -> Dict[str, Any]:
             add_if_exists(d, "split_feature", self._split_feature)
             add_if_exists(d, "threshold", self._threshold)
         else:
-            add_if_exists(d, "leaf_value", self._leaf_value)
+            if len(self._leaf_value) == 1:
```

Comment:
```
In [2]: len(foo)
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-2-63c8819f1adc> in <module>()
----> 1 len(foo)

TypeError: object of type 'NoneType' has no len()
```
We should protect when `self._leaf_value is None`

Reply:
I believe we're guarding against `None` a few lines above so should be good here?

Reply:
Yep, no worries here :D